src/usr/sbin/one-contextd

```diff
@@ -18,6 +18,8 @@
 
 set -e
 
+export LANG=C.UTF-8
```

Comment:
Could you elaborate why not just `LANG=C` - I can imagine that there could be an environment (custom busybox embedded something) where there is no other locale but POSIX C only.

Reply:
The `musl` libc use `C.UTF-8` [by default](https://wiki.musl-libc.org/functional-differences-from-glibc.html#Default-locale).

The Alpine is the only target I see that may have an issue with locales so I think `C.UTF-8` is a sane default.

Reply:
Hello @ospalax, is there concern for other cases than alpine linux?

Reply:
Hi @baby-gnu

sorry for being late with the reply - notification was lost on me in the inbox.

Anyway, regarding your comment - this file is shared across all distros - so this change affect all of them regardless of Alpine.

That is why `LANG=C` felt like more generic option - it is POSIX and valid. The fact that musl has different default change nothing about that.

I was thinking to modify your change to use `LANG=C` in front of all affected commands (where we parse the output in the pipe) instead of setting it globally for the whole `one-contexd ` and therefore respect the user's locale to print localized text where there is no issue with it.

So why I am talking about this - because I can imagine a system where is no support for locales (no unicode and no utf-8) - like this for example: https://github.com/antlr/antlr4/issues/1200

>  the Ubuntu docker images don't have any UTF-8 locales installed by default

`LANG=C` would still work though.

So I suggest to change it to `LANG=C` and secondary consider if we really want to set it globally or it would be nicer to set it only there where it really is needed (when we parse the output of some command).

Thanks